msal4j-sdk/src/test/java/com/microsoft/aad/msal4j/testcase/Act.java

```diff
@@ -0,0 +1,124 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.microsoft.aad.msal4j.testcase;
+
+import com.azure.json.JsonReader;
+import com.azure.json.JsonSerializable;
+import com.azure.json.JsonToken;
+import com.azure.json.JsonWriter;
+
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class Act implements JsonSerializable<Act> {
```

Comment:
I recommend you have a separate PR for this test infra.

Reply:
Removed in the latest commit, will make a new PR once we have a more official solution set up for these endpoints.